app/_common/components/NavigationBottom.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+
+import { navigate } from "../utils/redirect";
+import { Menubar, MenubarMenu, MenubarTrigger } from "../shadcn/ui/menubar";
+
+function NavigationBottom() {
+  const pathname = usePathname();
+
+  return (
+    <Menubar
+      defaultValue={pathname}
+      className="absolute bottom-8 left-2/4 z-50 -translate-x-2/4 shadow-neo-b"
```

Comment:
스크롤이 있는 페이지에서 사용하려면 fixed로 수정이 필요할 것 같습니다. 

```suggestion
      className="fixed bottom-8 left-2/4 z-50 -translate-x-2/4 shadow-neo-b"
```
![image](https://github.com/Open-Eye-Im-Developer/MoGakGo-FE/assets/86952779/80a3beb8-d24c-419d-911a-41ce04075753)